### ISSUE TITLE
All dates as date intervals.

### ISSUE THREAD

**Issue by DavidBrownlee** [CanDIG/MOHCCN_clinical_data_model#8]
Update multiple parts of the data model to reflect date intervals (rather than absolute dates).  (If this hasn't already been done...)

**Comment by kcranston** [comment]
Is this request for the files posted on the [MOHCCN polices page](https://www.marathonofhopecancercentres.ca/researcher-hub/policies-and-guidelines)? I think the aim was to keep dates in the spreadsheet, but describe the conversion to intervals in the guidelines.

**Comment by DavidBrownlee** [comment]
Yes, it was referring to the policies page.  We'll see what the data providers send in a few weeks and act accordingly.

**Comment by lstein** [comment]
DECISION (5 February 2025): This has previously been addressed in guidelines. Discussed distinguishing dummy dates at day resolution from dates at month resolution.
